src/util/dateHelpers.ts

```diff
@@ -0,0 +1,20 @@
+import Moment from 'moment';
+
+const getDateString = (date: Date): string => {
+    return Moment(date).format('DD.MM.YYYY');
+};
+
+const getDateWithoutHours = (date: Date): Date => {
```

Comment:
Rename to something like roundDateToMidnight?

Reply:
toMidnightDate seems to go better with toDateString